node_modules/oae-transfer/transfer/bundles/default.properties

```diff
@@ -0,0 +1,10 @@
+CANCEL_TRANSFER = Cancel the transfer
+CODE = Code 
+EMAIL = Email 
+ENTER_CODE = Enter the code...
+ENTER_EMAIL = Enter the email of your other account...
```

Comment:
Perhaps something more frienly and explicit here, such as "Fill in with the e-mail of the account you want to transfer to"